crl/updater/updater.go

```diff
@@ -330,6 +331,7 @@ func (cu *crlUpdater) tickShard(ctx context.Context, atTime time.Time, issuerNam
 		"Queried SA for CRL shard: id=[%s] numEntries=[%s]",
 		crlID, len(crlEntries))
 
+
```

Comment:
Merge introduced a double empty-line.

Reply:
Fixed, along with another fix I lost in the merge -- numEntries shouldn't be %s